### PR TITLE
Implement init and make associated updates for isNode === true

### PR DESCRIPTION
**Pull request by RNHTTR**
As discussed in telegram, I implemented `init` for `isNode === true`

`init` now accepts an optional `props` argument similar to `connect`. This allows a wallet to be initialized via a server rather than browser.

This required some downstream changes: There were calls to `this.activeWallet.account().accountId`, so I created a generic `getActiveAccountId()` method that got the accountId either via `activeWallet` or `activeAccount` depending on `isNode` & `isBrowser`.

### CODE REVIEW

File: src/wallet.ts [hunk 1]
```diff
@@ -277,18 +279,17 @@ export class Wallet {
...
-    if (!account || !accountId)
+    if (!accountId)
       return formatResponse({ error: 'Account is undefined.' })
 
     if (!keyPair || !accountId)
       return formatResponse({ error: `No Key Pair for account ${accountId}` })
 
     const publicKey = keyPair.getPublicKey().toString()
-    const balance = await account.getAccountBalance()
+    const balance = await this.activeAccount.getAccountBalance()
```
Comment (microchipgnu) [inline src/wallet.ts:292]: `this.activeAccount`might be undefined. 

```ts
this.activeAccount?.getAccountBalance()
```

File: src/wallet.ts [hunk 2]
```diff
@@ -359,15 +361,16 @@ export class Wallet {
...
+    if (!accountId)
+      return formatResponse({ error: 'AccountId is undefined.' })
+    if (!this.activeAccount)
+      return formatResponse({ error: 'No activeAccount set.' })
     if (!contractName)
-      return formatResponse({ error: 'No contract was provided.' })
+      return formatResponse({ error: 'No contract name was provided.' })
 
-    const contract = new Contract(account, contractName, {
+    const contract = new Contract(this.activeAccount, contractName, {
```
Comment (microchipgnu) [inline src/wallet.ts:373]: Every method that instantiates the contract with `this.activeAccount` errors out when called on the browser.

We were using `this.activeWallet?.account()` before.

Comment (RNHTTR) [reply src/wallet.ts:373]: Should there be isNode/isBrowser logic to use `this.activeAccount` or `this.activeWallet?.account()`, respectively?

Comment (RNHTTR) [reply src/wallet.ts:373]: bump :)

Comment (microchipgnu) [reply src/wallet.ts:373]: > Should there be isNode/isBrowser logic to use `this.activeAccount` or `this.activeWallet?.account()`, respectively?

Yes, I think that makes sense yes :)

Comment (microchipgnu) [reply src/wallet.ts:373]: @RNHTTR are you having any issues?

Comment (RNHTTR) [reply src/wallet.ts:373]: Hey, sorry! I've been a bit busy. I plan to try and implement the isNode/isBrowser logic hopefully this week.

Comment (microchipgnu) [reply src/wallet.ts:373]: No worries! Excited to take a look at it

Comment (RNHTTR) [reply src/wallet.ts:373]: @microchipgnu hey -- i'm not using mintbase anymore and haven't had a chance to get back around to this. I'd like to, but I'm not sure when. If you want to make the update instead, please feel free :)

**Pull request closed without merging**